org.gtk.Gtk3theme.Breeze.json

```diff
@@ -92,14 +92,24 @@
         {
           "type": "git",
           "url": "https://invent.kde.org/plasma/breeze-gtk.git",
-          "tag": "v5.24.1",
-          "dest": "breeze-gtk"
+          "tag": "v5.24.5",
+          "commit": "e630a07a28d675241e9c054699fd762baf13a800",
```

Comment:
Why commit if we already have tag? Seems duplicate info to me?

Reply:
ok, can you please change it to use tarballs + hashes then like we do in all the rest of KDE stuff? tarballs have always made more sense than git tags to me.

Reply:
Okay, done.